doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,312 @@
+==================================
+Gas-Solid Spouted Cylinder Bed
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters ind DEM and CFD-DEM.
+This is a modification of the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ . In this example, we deal with the geometry of a Cylinder. The boundary condition and some parameter have been changed accordingly.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a solid-gas cylinder shaped spouted bed
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-bed.prm``
```

Comment:
Same comment as above

Reply:
Also the file should the name generator in the name for it to be compatible with the CI for CFD-DEM examples.

Reply:
Sorry, I don't get it.
I changed into "dem-packing-in-spouted-cylinder-bed.prm" and "gas-solid-spouted-cylinder-bed.prm". Do they work?